packages/react-sdk/src/index.tsx

```diff
@@ -20,15 +20,22 @@ import {
   UnassignedFeedback,
 } from "@bucketco/browser-sdk";
 
+import { Features } from "../dev/nextjs-flag-demo/components/Flags";
```

Comment:
This is probably an incorrect import and why you are getting build errors?

Reply:
Thank you!!